packages/otelbin/src/components/monaco-editor/ErrorConsole.tsx

```diff
@@ -82,22 +82,22 @@ export default function ErrorConsole({ errors, font }: { errors?: IError; font:
 						{errors?.ajvErrors &&
 							errors.ajvErrors?.length > 0 &&
 							errors.ajvErrors.map((error: IAjvError, index: number) => {
-								return <Error key={index} error={error} font={font} />;
```

Comment:
**Issue:** The naming is a bit confusing, as this can also be a warning. Similar for the component `CountErrorMessages`